spec/requests/physical_servers_spec.rb

```diff
@@ -40,6 +49,33 @@
 
         expect(response).to have_http_status(:success)
       end
+
+      it "immediately restarts a server successfully" do
+        ps = FactoryGirl.create(:physical_server)
+
+        api_basic_authorize action_identifier(:physical_servers, :restart_now, :resource_actions, :post)
+        run_post(physical_servers_url(ps.id), gen_request(:restart_now))
+
+        expect(response).to have_http_status(:success)
```

Comment:
@abellotti I believe that the intent of these tests were to test that authorized actions succeed and unauthorized ones fail, so we were only really concerned about each call having a status of _success_ or _forbidden_ and not about the response details. I took a look at the responses of the _power_off_ action, as an example, and it appears that the various response fields differ between calls so simply passing a hash of expected values to _expect_ probably won't work.
```
{"success"=>true, "message"=>"Requested server state power_off for Server instance: 65000000000075 name:''", "task_id"=>"65r175", "task_href"=>"http://www.example.com/api/tasks/65r175", "href"=>"http://www.example.com/api/physical_servers/65r75"}
{"success"=>true, "message"=>"Requested server state power_off for Server instance: 65000000000091 name:''", "task_id"=>"65r183", "task_href"=>"http://www.example.com/api/tasks/65r183", "href"=>"http://www.example.com/api/physical_servers/65r91"}
{"success"=>true, "message"=>"Requested server state power_off for Server instance: 65000000000113 name:''", "task_id"=>"65r194", "task_href"=>"http://www.example.com/api/tasks/65r194", "href"=>"http://www.example.com/api/physical_servers/65r113"}
```